gui/components/navbar.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<body>
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container-fluid">
+      <!-- Brand and toggle get grouped for better mobile display -->
+      <div class="navbar-header">
```

Comment:
This is the bootstrap 3 navbar :)

Reply:
Keep bootstrap 3 for now, will switch to bootstrap 4 in the future.